remove_duplicates.py

```
@@ -0,0 +1,31 @@
+def remove_duplicates(input_file, output_file):
+    seen = set()
+    with open(input_file, 'r', encoding='utf-8') as infile, open(output_file, 'w', encoding='utf-8') as outfile:
+        for line in infile:
+            line = line.strip()
+            if not line:
+                continue
+            try:
+                data = json.loads(line)
+                # Sort keys to ensure consistent hashing
+                sorted_json = json.dumps(data, sort_keys=True)
+                hash_obj = hashlib.md5(sorted_json.encode('utf-8'))
```

Ruff (0.12.2)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)